lib/utils.pm

```diff
@@ -450,6 +450,9 @@ sub sle_version_at_least {
     my ($version, %args) = @_;
     my $version_variable = $args{version_variable} // 'VERSION';
 
+    my @valid_sle_versions = ('12', '12-SP1', '12-SP2', '12-SP3', '12-SP4', '15');
+    if (!grep $_ == get_var($version_variable), @valid_sle_versions) { return 0 }
```

Comment:
it should not use `==` but `eq` here. how about using

`return 0 unless grep {$_ eq $version_variable} @valid_sle_versions;`

Reply:
how about add a check in top of sle_version_at_least?
`return 0 unless check_var('DISTRI', 'sle')`

Reply:
@mitiao that was one of my first idea - but I'm sure this will mess up with caasp or similar other 'sle based' distris... hence I decided the other route... 

of course, if that is no concern, than that check is much simpler to maintain

Reply:
We verified and could indeed not find distri cases where this would be an issue - also, as those 'theoretical' cases would come with different versioning behind the scenes (e.g caasp is version 2.0 and would return true in any case already - which is wrong already if sle_at_least_version('12-sp1'( would be used)